src/ui.container-modal-confirm/index.vue

```diff
@@ -152,15 +153,20 @@ const props = defineProps({
 
 const emit = defineEmits(["update:modelValue", "confirm", "close"]);
 
+const { t } = useLocale();
+
 const {
-  config,
   hasSlotContent,
   footerLeftFallbackAttrs,
   modalAttrs,
   confirmButtonAttrs,
   cancelButtonAttrs,
 } = useAttrs(props);
 
+const customI18n = computed(() => {
+  return props.config.i18n || {};
```

Comment:
Why do we need to have computed here? And empy object always be present and we never to see `t('UModalConfirm.cancel')` in:

```vue
<UButton
          v-if="cancelButton"
          :label="customI18n.cancel || t('UModalConfirm.cancel')"
          variant="thirdary"
          filled
          :data-cy="`${dataCy}-close`"
```